modules/health_quest/app/services/health_quest/jwt_wrapper.rb

```diff
@@ -66,15 +66,15 @@ def gender
     end
 
     def dob
-      user.mpi&.profile&.birth_date
+      user.birth_date
```

Comment:
If needed we should alias user.dob to user.birth_date and try to document what a "user" is so we can update everyone to be using the same attributes.